osc/connection.py

```diff
@@ -733,7 +733,7 @@ def set_request_headers_after_401(self, url, request_headers, response):
             return False
 
         if not self.ssh_keygen_path:
-            output.print_msg("Skipping signature auth because ssh-keygen is not available", print_to="debug")
+            output.print_msg("Skipping signature auth because ssh-keygen is not available", print_to="warning")
```

Comment:
```suggestion
            output.print_msg("Skipping signature auth because ssh-keygen is not available", print_to="warning")
            output.print_msg("This can be ignored if you are not using SSH keys for authentication", print_to="warning")
```
Maybe a bit more of context for anyone still using passwords? Feel free to rework the message, or to just ignore this comment at all (at least with your change the situation is already better).